blocks/graph.py

```diff
@@ -354,3 +414,104 @@ def apply_noise(computation_graph, variables, level, seed=None):
         replace[variable] = (variable +
                              rng.normal(variable.shape, std=level))
     return computation_graph.replace(replace)
+
+
+def apply_dropout(computation_graph, variables, drop_prob=0.5, rng=None,
```

Comment:
I'm fine with removing it, asking the user to type 0.5 isn't too much of a burden, but the original paper does say

> We have tried various dropout probabilities and almost all of them improve the generalization
performance of the network. For fully connected layers, dropout in all hidden layers works
better than dropout in only one hidden layer and more extreme probabilities tend to be worse,
which is why we have used 0.5 throughout this paper.

And from Larochelle's course:

> Could use a different dropout probability, but 0.5 usually works well